pkg/api/types.go

```diff
@@ -2427,7 +2427,7 @@ type SerializedReference struct {
 type EventSource struct {
 	// Component from which the event is generated.
 	Component string `json:"component,omitempty"`
-	// Host name on which the event is generated.
+	// Node name on which the event is generated.
```

Comment:
Yes; I may be wrong, but I believe this is a Node.Name, not a host name.  If we are cross-referencing an event to a node, we should look up a Node.Name, not a host name.  We don't even publish the host name on a Node (except when it == the node name), so it would be wrong to publish with the hostname.

I don't think we can fix the json (that's schema), but we can at least fix the comment.